api/src/main/java/io/strimzi/api/kafka/model/status/ListenerStatus.java

```diff
@@ -28,19 +28,20 @@
         builderPackage = Constants.FABRIC8_KUBERNETES_API
 )
 @JsonInclude(JsonInclude.Include.NON_NULL)
-@JsonPropertyOrder({ "type", "addresses", "bootstrapServers", "certificates" })
+@JsonPropertyOrder({ "type", "name", "addresses", "bootstrapServers", "certificates" })
 @EqualsAndHashCode
 public class ListenerStatus implements UnknownPropertyPreserving, Serializable {
     private static final long serialVersionUID = 1L;
 
     private String type;
+    private String name;
     private List<ListenerAddress> addresses;
     private String bootstrapServers;
     private List<String> certificates;
     private Map<String, Object> additionalProperties;
 
-    @Description("The type of the listener. " +
-            "Can be one of the following three types: `plain`, `tls`, and `external`.")
+    @Description("The `type` field in `ListenerStatus` have been deprecated and will be removed when updating to CRD v1" +
+            "The type of the listener can be one of the following three types: `plain`, `tls`, and `external`.")
```

Comment:
This is not true and should be removed. You should instead say that this it the name of the listener.